src/chapters/radius-auth.xml

```diff
@@ -0,0 +1,240 @@
+            </varlistentry>
+            <varlistentry>
+                <term><property>radius-key-password</property></term>
+                <listitem>
+                    <para>The password of the private key specified in the 
+                        <property>radius-key-file</property> parameter.  Defaults to null if not specified.</para>
+                </listitem>
+            </varlistentry>
+            <varlistentry>
+                <term><property>ca-key-file</property></term>
+                <listitem>
+                    <para>The file that stores the certificate authority certificates for the
+                        connection to the RADIUS server.  Defaults to no file - if specified, the
+                        file must be present in the GUACAMOLE_HOME directory.</para>
+                </listitem>
+            </varlistentry>
+            <varlistentry>
+                <term><property>ca-key-type</property></term>
+                <listitem>
+                    <para>The type of file store used for the certificate authority.  If not
+                        specified this defaults to pem.</para>
+                </listitem>
+            </varlistentry>
+            <varlistentry>
+                <term><property>ca-key-password</property></term>
+                <listitem>
+                    <para>The password used to protect the certificate authority store, if
+                        any.  Default to null.</para>
+                </listitem>
+            </varlistentry>
+            <varlistentry>
+                 <term><property>radius-trust-all</property></term>
+                 <listitem>
+                     <para>Trust all server certificates without verifying against a CA file.
+                     </para>
+                 </listitem>
+            </varlistentry>
+            <varlistentry>
+                <term><property>radius-retries</property></term>
+                <listitem>
+                    <para>The number of times the client will retry the connection to the 
+                        RADIUS server and not receive a response before giving up.  The default
+                        is 5.</para>
+                </listitem>
+            </varlistentry>
+            <varlistentry>
+                <term><property>radius-timeout</property></term>
+                <listitem>
+                    <para>The timeout for a RADIUS connection in seconds.  The default is
+                        60 seconds.</para>
+                </listitem>
+            </varlistentry>
+            <varlistentry>
+                <term><property>radius-eap-ttls-inner-protocol</property></term>
+                <listitem>
+                    <para>When EAP-TTLS is used, this parameter specifies the inner (tunneled)
+                        protocol to use talking to the RADIUS server.  It is required when the
+                        <property>radius-auth-protocol</property> parameter is set to eap-ttls.</para>
```

Comment:
What is the behavior of this parameter if `radius-auth-protocol` is *not* set to `eap-ttls`? If it has no effect, we should note that here.

Reply:
Fixed.